fastani/1.34-RGDV2/Dockerfile

```diff
@@ -0,0 +1,76 @@
+## build RGDv2 ##
+FROM staphb/ncbi-datasets:latest as stage
+
+# copy in list of NCBI accessions and species list
+COPY RGDv2-NCBI-assembly-accessions.txt /RGDv2/RGDv2-NCBI-assembly-accessions.txt
+COPY RGDv2-NCBI-assembly-accessions-and-species.txt /RGDv2/RGDv2-NCBI-assembly-accessions-and-species.txt
+
+# download RGD genomes using NCBI datasets tools; cleanup unneccessary files; 
+# move and re-name assemblies to include Species in the filename
+# make fasta files readable to all users; create File Of FileNames for all 43 assemblies (to be used with fastANI)
+RUN for ID in $(cat /RGDv2/RGDv2-NCBI-assembly-accessions.txt); do \
+ SPECIES=$(grep "${ID}" /RGDv2/RGDv2-NCBI-assembly-accessions-and-species.txt | cut -f 1) && \
+ echo "downloading $ID, species "${SPECIES}", from NCBI..."; \
+ datasets download genome accession ${ID} --filename ${ID}.zip; \
+ unzip ${ID}.zip; \
+ rm ${ID}.zip; \
+ mv -v ncbi_dataset/data/${ID}/${ID}*.fna /RGDv2/${ID}.${SPECIES}.fasta; \
+ rm -rfv ncbi_dataset/; \
+ rm -v README.md; \
+done && \
+ls /RGDv2/*.fasta >/RGDv2/FOFN-RGDv2.txt &&\
+chmod 664 /RGDv2/*
+
+## App ##
+FROM ubuntu:jammy as app
+
+# for easy upgrade later. ARG variables only persist at build time
+ARG FASTANI_VER="v1.34"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="FastANI"
+LABEL software.version=${FASTANI_VER}
+LABEL description="Fast alignment-free computation of whole-genome Average Nucleotide Identity"
+LABEL website="https://github.com/ParBLiSS/FastANI"
+LABEL license="https://github.com/ParBLiSS/FastANI/blob/master/LICENSE"
+LABEL maintainer="Kelsey Florek"
+LABEL maintainer.email="kelsey.florek@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Kutluhan Incekara"
+LABEL maintainer3.email="kutluhan.incekara@ct.gov"
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y wget \
+ unzip \
+ libgomp1 && \
+ apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
What happens if you don't install recommended packages? (--no-install-recommends)
```
RUN apt-get update && apt-get install -y --no-install-recommends \
   unzip \
   libgomp1 && \
   apt-get autoclean && rm -rf /var/lib/apt/lists/*
   ```

Reply:
Oh, I missed it. Let me check it.